script.js

```diff
@@ -1,22 +1,32 @@
 function poundsToDollars(amount) {
-	if (typeof amount == 'string') {
+	if (typeof amount === 'string') {
```

Comment:
Change this to  	if (isNaN(amount)) {
		throw new Error('Error: You havent entered a number');
	} because checking if it a string will throw an error on the user interface